frontend/src/routes/Governance/policies/policy-details/PolicyTemplateDetails.tsx

```diff
@@ -82,19 +98,52 @@ export function PolicyTemplateDetails() {
       },
       {
         key: t('Cluster'),
-        value: template ? clusterName : '-',
+        value: clusterName || '-',
       },
       {
         key: t('Kind'),
         value: kind ?? '-',
       },
       {
         key: t('API groups'),
-        value: template?.apiVersion ?? '-',
+        value: apiVersion ? apiGroup + '/' + apiVersion : apiGroup,
       },
     ]
 
-    if (template?.spec?.match?.kinds) {
+    if (apiGroup === 'constraints.gatekeeper.sh') {
+      // Loading to fetch VAPB
+      if (vapb.loading && !vapb.vapbObj) {
+        cols.push({
+          key: '',
+          keyAction: (
+            <Skeleton width="100px" height="60px" screenreaderText="Fetching ValidatingAdmissionPolicyBinding" />
```

Comment:
Could you please always show the row with the header "Validating Admission Policy Binding" and only show the skeleton while the query is going? If there's no VAPB then I think a value of `-` or `Not applicable` would be good.